ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/_cq_dialog/.content.xml

```diff
@@ -151,6 +151,19 @@
                                                         name="./default"
                                                         required="{Boolean}false"/>
                                             </defaultValueMS>
+                                            <autoComplete
+                                                    granite:class="cmp-adaptiveform-dropdown__autocomplete"
+                                                    jcr:primaryType="nt:unstructured"
+                                                    sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                                    name="./fd:autoComplete"
+                                                    text="Enable Auto-Completion"
```

Comment:
please get these strings reviewed by CHL team. We might also want to add description here via information icon